src/pages/en/guides/cms/wordpress.md

```diff
@@ -11,8 +11,9 @@ service: WordPress
 ## Community Resources 
 
 - [Building An Astro Website With WordPress As A Headless CMS](https://blog.openreplay.com/building-an-astro-website-with-wordpress-as-a-headless-cms/) by Chris Bongers.
-
 - [Building with Astro x Wordpress](https://www.youtube.com/watch?v=Jstqgklvfnc) on Ben Holmes's stream.
+- [Building a Headless WordPress Site with Astro](https://developers.wpengine.com/blog/building-a-headless-wordpress-site-with-astro) by Jeff Everhart
+- You can get started with this [Astro WordPress Starter](https://github.com/JEverhart383/astro-wordpress-starter)
```

Comment:
```suggestion
- [Building a Headless WordPress Site with Astro](https://developers.wpengine.com/blog/building-a-headless-wordpress-site-with-astro) by Jeff Everhart
```
So happy to have this in our docs! I double-checked first, and since the starter repo is linked to from the article, we'll go with just the article here. People will be able to make their way to the repo, (they'll HAVE to read your blog post to get to the repo...) and we get to showcase your helpful content!  ;)